javascripts/controllers/EditCtrl.js

```diff
@@ -12,7 +12,7 @@ app.controller("EditCtrl", function($location, $routeParams, $scope, $rootScope,
     getContact();
 
     $scope.updateContactInFirebase = (contact, contacts) => {
```

Comment:
why are you passing contacts in here?  do you need contact and contacts

Reply:
I need contact. I deleted contacts!